src/graphql/resolvers/monsterResolver.ts

```diff
@@ -69,22 +83,22 @@ const Monster = {
       proficiency: profs.find(prof => prof.index === p.proficiency.index),
     }));
   },
-  reactions: async monster =>
+  reactions: async (monster: Monster) =>
     monster.reactions
       ? monster.reactions.map(async r => {
-          const resolvedReaction = { ...r };
+          const resolvedReaction: Record<any, any> = { ...r };
```

Comment:
`any` shouldn't be the record key.

Reply:
I think there are a few more uses of `any` as a record key in this file.

Reply:
I cleaned up a bunch more in other files too. Thanks for calling me out on that.